frontend/src/pages/application/createInstance.vue

```diff
@@ -130,6 +130,7 @@ export default {
         createInstance (instanceDetails, copyParts) {
             const createPayload = { ...instanceDetails, applicationId: this.application.id }
             if (this.sourceInstance?.id) {
+                delete createPayload.flowBlueprintId
```

Comment:
It feels like this shouldn't be there in the first place, but if this is the first spot to remove it! 👍

Reply:
It looks like it's initialised to an empty string which then gets mapped to the default in the backend.

So it's either strip it here or add a test in the backend.